data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+			choice
+				`	"I had heard about the kidnapping."`
+			`	He nods, "Unfortunately, it's taking a while for them to redistribute an appropriately resourced defence force thanks to the Unfettered."`
+			`	"Even more unfortunately," he continues, "it seems that the overabundance of these pirates is a result of one particular individual's success. His ship, called the Black Lich, is hanging out in Waypoint with the stolen Geocoris that we hear might have been converted into some kind of semi-stable weapons platform and now be being used as a temporary base."`
+			`	"Fortunately this gives us an opportunity though. If the kidnapped Hai are still on board we might be able to rescue them, and if we can defeat the Black Lich we may stem the tide of opportunists that are swarming through the wormhole."`
+			`	"What do you say?"`
+			`	The poor Captain seems desperate.`
+			choice
+				`	"We shall bring them safely home."`
+					accept
+	npc
+		government "Hai"
+		personality escort heroic
+		fleet "Anti-Pirate Hai (No Wormhole Travel)" 5
+	npc
+		government "Pirate"
+		personality staying disables plunders harvests
+		system "Waypoint"
+		fleet "Marauder fleet IX" 4
+		fleet "Hired Guns" 1
+	npc kill
+		government "Pirate"
+		personality staying marked target
+		system "Waypoint"
+		ship "Marauder Leviathan (Weapons)" "Black Lich"
+	to complete
+		or
+			has "00 Hai Leaks Response 4B: declined"
+			has "Hai Leaks Response 4A: failed"
+	on complete
+		payment 1000000
+		event "waypoint pirates vanquished"
+		log `Defeated a nasty marauder whose ship was the "Black Lich". He had managed to kidnap a number of Hai straight off the planet with a combination of trickery and speed. You were rewarded well for this particular piece of aid.`
+		conversation
+			`When you return the Hai Captain thanks you profusely, "Please, the Allhome government has authorised a reward of <payment> for routing the problem."`
+			branch "saved geocoris"
+				has "00 Hai Leaks Response 4B: declined"
+			`	He sighs deeply, "It is a shame though that the kidnapped Hai could not be saved, but the other Hai captains tell me it was a mess out there."`
+			`	He places one of his furry hands comfortingly on your shoulder, "I'm sure you did all you could."`
+				goto end
+			label "saved geocoris"
+			`	"Your rescue of our kidnapped people is deeply appreciated also. Lucky for you, the merchant who owned the Geocoris has already filed the insurance claim, so you get to keep the ship as a trophy."`
+			`	"I do need to do a full security scan of it before I let it go though, just to complete all the incident paperwork, so if you have any business on world here then you should take your time."`
```

Comment:
It's odd to imply that there will be some significant wait when the player can just immediately leave after this conversation. Maybe add a little on at the end of what you have noting that `<first>` waits for a few hours or something?

Reply:
It's more meant to be a "if you wanted to play with outfits and stuff, now is a good time" moment. I see your point though.